WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**
+N.B. The command line commands in this text file work for Mac but may not work for Windows and Linux. You might need to look up the equivalent commands for these operating systems.
+
+HOW TO SET UP WATERDOCK-2.0 (DETAILED)
+
+1.   Download the zip file or clone using GitHub.
+
+
+2.   Have Python 3.7 (or the latest version of Python 3 installed onto your computer).

```

Comment:
It probably would be sufficient to just say 3.7+ here.

Reply:
Yes it would, I have changed it.